assets/queries/openAPI/success_response_code_undefined_head_operation/query.rego

```diff
@@ -0,0 +1,22 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+	response := doc.paths[n][oper].responses
+
+	oper == "head"
```

Comment:
Since we are checking this for more than one query, what do you think about creating a generic function to receive response or undefined?

Reply:
Also, I think [same comment applies here](https://github.com/Checkmarx/kics/pull/3097#discussion_r625865245)